### ISSUE TITLE
Failing to copy dylib on Mac when building from command line

### ISSUE THREAD

**Issue by patchuby** [YoYoGames/GMEXT-EpicOnlineServices#7]
The build succeeds when launching from the IDE but not from the command line.
The command line also works for Windows.
libEOSSDK-Mac-Shipping.dylib is in the correct place.

```
[EPICONLINESERVICES] INIT: Script initialization succeeded (v1.4.1 :: 1).
"Copying macOS (64 bit) dependencies"
New-Item : Cannot bind argument to parameter 'Path' because it is an empty string.
At line:1 char:43
+ ... Force -Path (Split-Path -Parent '\\Supporting Files\libEOSSDK-Mac-Shi ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [New-Item], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorEmptyStringNotAllowed,Microsoft.PowerShell.Commands.NewI 
   temCommand
 
Copy-Item : The network path was not found
At line:1 char:115
+ ... ng.dylib'); Copy-Item -Path 'C:\CDS-Assets\export_config\gm_building\ ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Copy-Item], IOException
    + FullyQualifiedErrorId : System.IO.IOException,Microsoft.PowerShell.Commands.CopyItemCommand
 
[EPICONLINESERVICES] ERROR: Failed to copy 'C:\CDS-Assets\export_config\gm_building\epic_game_store\sdk\Bin\libEOSSDK-Mac-Shipping.dylib' to '\\Supporting Files\libEOSSDK-Mac-Shipping.dylib'.
```

**Comment by DiasFranciscoA** [comment]
This issue as now been fix in the lastest version v1.4.2

**Comment by patchuby** [comment]
Confirmed, no issues anymore when building
